src/Menu/MenuBuilder.php

```diff
@@ -56,6 +58,17 @@ public function resolve(array $menu)
         return $this->menuBuilder($menu);
     }
 
+    /**
+     * Here to allow transition between old and new query engines. For internal use only.
+     *
+     * @internal
+     * @param $accessor
+     */
+    public function setQueryEngine($accessor)
```

Comment:
I really don't think we need this, as we can just modify the constructor signature to add `$accessor = 'storage`

Reply:
Updated